Sources/StreamChat/Models/Channel.swift

```diff
@@ -248,11 +248,16 @@ extension ChatChannel: Hashable {
 /// A struct describing unread counts for a channel.
 public struct ChannelUnreadCount: Decodable, Equatable {
     /// The default value representing no unread messages.
-    public static let noUnread = ChannelUnreadCount(messages: 0, mentionedMessages: 0)
+    public static let noUnread = ChannelUnreadCount(messages: 0, mentioningMessages: 0)
     
     /// The total number of unread messages in the channel.
-    public internal(set) var messages: Int
+    public let messages: Int
     
     /// The number of unread messages that mention the current user.
-    public internal(set) var mentionedMessages: Int
+    public let mentioningMessages: Int
+}
+
+public extension ChannelUnreadCount {
+    @available(*, deprecated, renamed: "mentioningMessages")
+    var mentionedMessages: Int { mentioningMessages }
```

Comment:
Is the renaming really necessary? The previous one is already fine IMO 🤔 Not sure if it is worth deprecating

Reply:
The current naming is confusing, IMO. It's not clear what are these mentioned messages, who has mentioned them and why we're interested in this number 😄 Whereas `mentioning` is more clear because it states that some of the unread messages mention the current user.

Reply:
Done ✅